test/e2e/kubevirt.go

```diff
@@ -2253,11 +2268,40 @@ chpasswd: { expire: False }
 			by(vmi.Name, "Running live migration for virtual machine instance")
 			td(vmi)
 
-			step = by(vmi.Name, fmt.Sprintf("Login to virtual machine after virtual machine instance live migration"))
+			// Update vmi status after live migration
+			Expect(crClient.Get(context.Background(), crclient.ObjectKeyFromObject(vmi), vmi)).To(Succeed())
+
+			step = by(vmi.Name, "Login to virtual machine after virtual machine instance live migration")
 			Expect(kubevirt.LoginToFedora(vmi, "fedora", "fedora")).To(Succeed(), step)
 
 			step = by(vmi.Name, "Check east/west traffic after virtual machine instance live migration")
 			checkEastWestIperfTraffic(vmi, testPodsIPs, step)
+
+			By("Stop iperf3 traffic before force killing vm, so iperf3 server do not get stuck")
+			output, err = kubevirt.RunCommand(vmi, "killall iperf3", 5*time.Second)
+			Expect(err).ToNot(HaveOccurred(), output)
+
+			step = by(vmi.Name, fmt.Sprintf("Force kill qemu at node %q where VM is running on", vmi.Status.NodeName))
+			output, err = infraprovider.Get().ExecK8NodeCommand(vmi.Status.NodeName, []string{"bash", "-c", `kill -9 $(pidof virt-launcher)`})
+			Expect(err).ToNot(HaveOccurred(), output)
+
+			step = by(vmi.Name, "Waiting for failed restarted VMI to reach ready state")
+			waitVirtualMachineInstanceFailed(vmi)
+			waitVirtualMachineInstanceReadiness(vmi)
+			Expect(crClient.Get(context.TODO(), crclient.ObjectKeyFromObject(vmi), vmi)).To(Succeed())
+
+			step = by(vmi.Name, "Login to virtual machine after virtual machine instance force killed")
+			Expect(kubevirt.LoginToFedora(vmi, "fedora", "fedora")).To(Succeed(), step)
+
+			step = by(vmi.Name, "Restart iperf traffic after forcing a vm failure")
+			Expect(startEastWestIperfTraffic(vmi, testPodsIPs, step)).To(Succeed(), step)
+			checkEastWestIperfTraffic(vmi, testPodsIPs, step)
+
+			by(vmi.Name, "Running live migration after forcing vm failure")
+			td(vmi)
+
+			step = by(vmi.Name, "Check east/west traffic for failed virtual machine after live migration")
+			checkEastWestIperfTraffic(vmi, testPodsIPs, step)
```

Comment:
Why do we need to check migration again?
Isnt connectivity check is good enough indication that the network plumbing is in good shape?

Reply:
> > Why do we need to check migration again?
> 
> Good question :)
> 

As said without another migration we do not reproduce the issue where the VM is moved to the node where the danling LSP is.

> > Isnt connectivity check is good enough indication that the network plumbing is in good shape?
> 
> I would have been even lazier, and just test that the VM starts up again, and goes to ready state. But I agree that checking traffic is better (but might be redundant, since we test vm startup elsewhere).

Checking VM start will prove nothing, we either check danling LSPs direclty with ovn-nbctl (that's not right from integration tests) or we test what we know is failing wich is the traffic.

Reply:
I thought the pods failed to start .

Reply:
> I thought the pods failed to start .

Problem is that traffic fails since the same logical switch has two ports with the same mac and one lsp up=true and the other up=false